packages/terra-cell-grid/tests/wdio/cell-grid-spec.js

```diff
@@ -1,16 +1,16 @@
 Terra.describeViewports('CellGrid', ['medium'], () => {
-  describe('Cell Test', () => {
-    before(() => browser.url('/#/raw/tests/terra-cell-grid/cell-grid/cell'));
-    Terra.it.validatesElement();
+  it('Cell test', () => {
```

Comment:
`it` blocks aren't really a replacement for `describe` blocks. `it` blocks should describe the action or assertion the block is testing.

```js
it('should render a grid cell', () => {
  // Steps and assertions.
});
```

Reply:
Updated here https://github.com/cerner/terra-core/pull/3231/commits/85f452662ab0602ace7b2b5c36b71896167f5733